data-source/jsonnet/northern-ireland/blocks/individual/identity-and-health/arrive_in_country.jsonnet

```diff
@@ -35,4 +35,24 @@ local proxyTitle = {
       when: [rules.proxyYes],
     },
   ],
+  routing_rules: [
+    {
+      goto: {
+        block: 'passports',
+        when: [
+          {
```

Comment:
```
         {
            id: 'country-of-birth-answer',
            condition: 'equals',
            value: 'Northern Ireland',
          },
```
You can't reach this block if you're from NI, so this rule (under 1s should skip one-year-ago questions) won't apply to anyone. The above block needs removing.

Reply:
Removed.